code/modules/hydroponics/seed_datums.dm

```diff
@@ -184,7 +184,7 @@
 	display_name = "killer tomato plant"
 	mutants = null
 	can_self_harvest = 1
-	has_mob_product = /mob/living/simple_animal/tomato
+	has_mob_product = /mob/living/simple_animal/hostile/tomato
```

Comment:
I spoke with someone and they said that tomatos do attack you when created, so it made sense to subtype them. In response to your earlier question about why I have the paths, that's purely for admin spawning or mapping ease. If you want hostile mobs, you can just look at the path. That's all. It's not necessary to leave it in if you don't like it, but it's sort of nice to know at a glance what your choices are when mapping for "hostile" things.